fractal_server/app/routes/admin/v2.py

```diff
@@ -307,3 +314,126 @@ async def flag_task_v1_as_v2_compatible(
     await db.close()
 
     return Response(status_code=status.HTTP_200_OK)
+
+
+class TaskV2Minimal(BaseModel):
```

Comment:
Let's start with a narrow-scoped model (we can always add attributes in the future if needed).
I would include: id, name, type, commands, source, owner, version (thus removing metas, docs, I/O types).